nixos/modules/services/desktop-managers/lomiri.nix

```diff
@@ -0,0 +1,165 @@
+{ config, pkgs, lib, ... }:
+
+let
+  cfg = config.services.desktopManager.lomiri;
+in {
+  options.services.desktopManager.lomiri= {
+    enable = lib.mkEnableOption (lib.mdDoc ''
```

Comment:
`lib.mdDoc` was removed on master.

Reply:
Ah right, I saw https://github.com/NixOS/nixpkgs/pull/303841 yesterday but forgot to subscribe to it & address the removal here. Thanks!